### ISSUE TITLE
Missing reference to MAPIT-tools repo necessary for "Getting Started"

### ISSUE THREAD

**Issue by gonuke** [sandialabs/MAPIT#11]
The tutorial information on the MAPIT documentation refers to files that don't exist in the `MAPIT` repo and there is no reference to the `MAPIT-tools` repo that appears to contain that info.  It would be helpful for your install guide info to point new users to that repo to help get started smoothly.

**Comment by gonuke** [comment]
I wonder if the documentation was generated from the files in the `pm-bugfixes` branch?  That might explain the discrepancy - that branch appears to add 1500 lines of code and many files.

**Comment by nshoman** [comment]
Thanks for mentioning this. I closed that PR as it was stale and fixed in a separate commit. 

In short, we used to include many auxiliary files in MAPIT (example datasets, install scripts, etc) that we removed and split off into mapit-tools (https://github.com/sandialabs/MAPIT-tools). Reason being, we are trying to have this repo serve as just the core of MAPIT and also as feedstock for a conda-forge recipe. That's seen some setbacks so the documentation got stale. 

tl;dr - this repo will work properly, but doesn't include extras. Launching MAPIT should prompt you to point towards the extra data from MAPIT-tools (if you want to use it). I'll try to update the docs to a minimal degree early next week (we've made some large API changes we are hoping to incorporate soon, but there's some license issues we are working out).

**Comment by nshoman** [comment]
I pushed a new release (1.3.35-beta) that fixed the `setup.py` so you should be able to install directly from the repo using `pip` under most circumstances (except for Apple silicon). 

We've updated the documentation in a105f1a8e611a123ad87391b3031ebcdab164a6a as well.

There's some inconvenient placement of output files as a result of the fixed `setup.py` (output located in site-packages) that we will be updating in a future update.
